### PR TITLE
Use sigHandler() as unique signal handler

### PR DESCRIPTION
**Pull request by eiffel-fl**
Hi.

In this PR, I removed the `sigs` channel in `genericStreams()`, thus all the signals are now handled in `sigHandler`.
This prevents having race condition regarding which channel will receive the signal first.

Best regards.

### CODE REVIEW

File: cmd/kubectl-gadget/utils/trace.go [hunk 1]
```diff
@@ -1037,7 +1037,12 @@ func genericStreams(
 	for {
 		select {
 		case msg := <-completion:
-			fmt.Fprintln(os.Stderr, msg)
+			output := os.Stdout
+			if strings.HasPrefix(msg, "Error") {
+				output = os.Stderr
+			}
+			fmt.Fprintln(output, msg)
+
```
Comment (mauriciovasquezbernal) [inline cmd/kubectl-gadget/utils/trace.go:1045]: It seems to hacky to me. I'd rather remove the "completed" message than having this logic here.

File: cmd/kubectl-gadget/utils/trace.go [hunk 2]
```diff
@@ -701,7 +701,7 @@ var sigIntReceivedNumber = 0
 // On reception of this signal, the given trace will be deleted.
 // This function fixes trace not being deleted when calling:
 // kubectl gadget process-collector -A | head -n0
-func sigHandler(traceID *string) {
+func sigHandler(traceID *string, willPrintTerminatingMessage bool) {
```
Comment (mauriciovasquezbernal) [inline cmd/kubectl-gadget/utils/trace.go:704]:
```suggestion
func sigHandler(traceID *string, printTerminatingMessage bool) {
```

?

**Review by mauriciovasquezbernal — APPROVED** [review]
This change LGTM, it fixes the problem that we have because we're adding multiple handlers for SIGINT.

File: cmd/kubectl-gadget/utils/trace.go (hunk 2 above)
Comment (blanquicet) [reply cmd/kubectl-gadget/utils/trace.go:704]: `printTerminationMessage`? 😆

**Review by blanquicet — APPROVED** [review]
LGTM!

**Pull request merged**